drivers/SmartThings/zigbee-switch/profiles/switch-power-2.yml

```diff
@@ -10,13 +10,5 @@ components:
         version: 1
       - id: refresh
         version: 1
-    categories:
-      - name: SmartPlug
-  - id: switch1
-    capabilities:
-      - id: switch
-        version: 1
-      - id: powerMeter
-        version: 1
```

Comment:
It might be more useful to change the profile the fingerprint points to, since deleting this component and still having it called "switch power 2" doesn't make too much sense. We can't move or delete it because Sprocket will throw an error, but this profile is probably useful for some other device later.

Reply:
@greens done